### PR TITLE
Rc/0.3.0

### PR DESCRIPTION
**Pull request by tomkad99**
image

### CODE REVIEW

File: tool/analyzer_plugin/bin/plugin.dart
Comment (LukaszWojcikIteo) [inline tool/analyzer_plugin/bin/plugin.dart:1]: Package structure is defined by analyzer_plugin

https://github.com/dart-lang/sdk/blob/main/pkg/analyzer_plugin/doc/tutorial/package_structure.md

File: tool/analyzer_plugin/pubspec.yaml
Comment (LukaszWojcikIteo) [inline tool/analyzer_plugin/pubspec.yaml:1]: Change architecture_linter dependency in this pubspec to be consistent with the plugin version

**Review by LukaszWojcikIteo — CHANGES_REQUESTED** [review]

File: tool/analyzer_plugin/bin/plugin.dart
Comment (tomkad99) [reply tool/analyzer_plugin/bin/plugin.dart:1]: Done

File: tool/analyzer_plugin/pubspec.yaml
Comment (tomkad99) [reply tool/analyzer_plugin/pubspec.yaml:1]: Done

**Review by LukaszWojcikIteo — APPROVED** [review]
LGTM

**Pull request merged**